microprofile/WFLY-19592_MicroProfile_Fault_Tolerance_4_1.adoc

```diff
@@ -0,0 +1,127 @@
+---
+categories:
+- microprofile
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/626
+feature-team:
+ developer: rhusar
+ sme:
+  - rhusar
+ outside-perspective:
+  -
+promotes:
+promoted-by:
+---
+= WFLY-19592 [MP Platform 7] MicroProfile Fault Tolerance 4.1
+:author:            Radoslav Husar
+:email:             rhusar@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Goal is to integrate MicroProfile (MP) Fault Tolerance (FT) 4.1 specification from the MP Platform 7.0 to WildFly at default stability level.
+To certify functionality, the MP FT TCK is run as part of the WildFly testsuite.
+
+=== User Stories
+
+User wants to deploy an application using MicroProfile Fault Tolerance 4.1 from MicroProfile Platform 7.0.
+
+== Issue Metadata
+
+* https://issues.redhat.com/browse/WFLY-19592[WFLY-19592 [MP Platform 7] MicroProfile Fault Tolerance 4.1]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-19592[WFLY-19588 Implement MicroProfile Platform 7] – MP Platform parent issue
+* https://issues.redhat.com/browse/WFLY-19590[WFLY-19590 MicroProfile Telemetry 2.0] – dependency of this issue for full Telemetry metrics integration
+
+// EAP Tracking issue:
+// * https://issues.redhat.com/browse/EAP7-2225[EAP7-2225 [MP Platform 7] MicroProfile Fault Tolerance 4.1]
+
+=== Affected Projects or Components
+
+* Eclipse MicroProfile Fault Tolerance
+* SmallRye Fault Tolerance
+
+=== Other Interested Projects
+
+* N/A
+
+=== Relevant Installation Types
+
+* Traditional standalone server (unzipped or provisioned by Galleon)
+* OpenShift Source-to-Image (S2I)
+* Bootable jar
+
+== Requirements
+
+* Upgrade MP FT specification modules to 4.1.
+* Upgrade SmallRye FT to 6.5.x.
+* Enable integration with MP Telemetry Metrics when the subsystem is available in the serve configuration.
+
+=== Changed requirements
+
+* N/A
+
+=== Non-Requirements
+
+* N/A
+
+=== Future Work
+
+None.
+
+== Backwards Compatibility
+
+The specification and the implementation do not introduce any incompatibilities from the previous version (MP FT 4.0).
```

Comment:
Question: will users having code in place that integrates MP Metrics be able to run it?